src/librustdoc/clean/types.rs

```diff
@@ -1339,13 +1339,14 @@ impl GenericParamDefKind {
 
 #[derive(Clone, PartialEq, Eq, Debug, Hash)]
 pub(crate) struct GenericParamDef {
+    pub(crate) did: DefId,
```

Comment:
Nitpicking: please use `def_id`. We tend to use this over `did` in rustdoc as much as we can and I'd like this trend to continue. :)